actionlist/app/containers/PreferencesContainer.js

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native';
+import Button from 'react-native-button';
+import { connect } from 'react-redux';
+import { Actions } from 'react-native-router-flux';
+import ModalDropdown from 'react-native-modal-dropdown';
+import Header from '../components/header';
+import { selectDropdownOption } from '../actions/action_items';
+import { Colors } from '../lib/commons';
+
+const styles = StyleSheet.create({
+  componentContainer: {
+    backgroundColor: '#808080',
+    marginTop: 10,
+    marginRight: 10,
+    marginLeft: 10,
+    padding: 10,
+    alignItems: 'center',
+    borderRadius: 8,
+    borderStyle: 'solid',
+    borderWidth: 1,
+
+  },
+  text: {
+    color: Colors.IUCrimson,
+    marginLeft: 10,
+  },
+  preferencesContainer: {
+    marginTop: 10,
+    backgroundColor: 'grey',
+    marginRight: 10,
+    marginLeft: 10,
+    borderRadius: 8,
+    borderStyle: 'solid',
+    borderWidth: 1,
+  },
+  preferenceSelect: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    marginBottom: 13,
+    marginLeft: 20,
+    marginRight: 10,
+    marginTop: 12,
+  },
+  preferenceText: {
+    fontSize: 15,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  preferences_buttons: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    marginTop: 10,
+    marginLeft: 10,
+    marginRight: 10,
+    justifyContent: 'space-between',
+  },
+
+  buttonContainer: {
+    borderRadius: 8,
+    width: 110,
+  },
+  dropdown: {
+    width: 150,
+    borderStyle: 'solid',
+    borderWidth: 1,
+    marginRight: 30,
+  },
+  preferencesButton: {
+    borderRadius: 8,
+    backgroundColor: '#990000',
+    marginRight: 10,
+    marginLeft: 10,
+    padding: 5,
+    fontSize: 15,
+    color: '#ffffff',
+    height: 30,
+    alignItems: 'center',
+  },
+});
+
+const OPTIONS = [
+  'aqua',
+  'blue',
+  'green',
+  'orange',
+  'pink',
+  'purple',
+  'red',
+  'grey',
+  'tan',
+  'white',
+];
+
+const PreferencesContainer = ({ onSelectOption, dropdownColors }) => (
+  <View>
+    <Header />
+    <View style={styles.componentContainer}>
+      <Text style={styles.preferenceText}>Preferences</Text>
+    </View>
+    <View style={styles.preferencesContainer}>
+      <View style={styles.preferenceSelect}>
+        <View>
+          <Text style={styles.text}>Saved</Text>
+        </View>
+        {/* Disabled Following ES-Lint rule
+          It is corrected in latest version which is still in beta */}
+        <View>
```

Comment:
These ModalDropdown views can probably be extracted to a utility component that takes the index, value, and document status properties so you're not repeating so much code.